examples/demo_rcv/src/main.rs

```diff
@@ -253,23 +253,27 @@ fn handle_input(dmx_recv: &mut SacnReceiver) -> Result<bool> {
                 ACTION_SLEEP => {
                     if split_input.len() < 2 {
                         display_help();
-                        bail!(std::io::Error::new(std::io::ErrorKind::InvalidInput, "Insufficient parts ( < 2 )"));
+                        return Err(std::io::Error::new(std::io::ErrorKind::InvalidInput, "Insufficient parts ( < 2 )"));
```

Comment:
All the insufficient parts errors can be their own error type so we can get rid of the human readable message